### DISCUSSION TITLE
"Types of changes" don't account for documentation fixes

### DISCUSSION THREAD

**Discussion by malaire** [olivierlacan/keep-a-changelog#506] [Q&A]
Following types of changes are supported (from https://keepachangelog.com):

>Types of changes
> - Added: for new features.
> - Changed: for changes in existing functionality.
> - Deprecated: for soon-to-be removed features.
> - Removed: for now removed features.
> - Fixed: for any bug fixes.
> - Security: in case of vulnerabilities.

None of these is suitable for documentation fixes.

**Comment by blaisep-vgs** [comment]
FWIW, @malaire , I use Sphinx to build my docs and treat my documentation as code, so I apply the term "feature" to include documentation.
